docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,177 @@
+nav: 12
+---
+
+## Introduction
+
+Virtual background plugin helps in customising one’s background i.e. replacing the background with a static image.
+This guide provides an overview of usage of the virtual background plugin of 100ms.
+
+![VirtualBackground](/docs/v2/vb.jpg)
+
+## Supported Android API Levels
+
+Virtual background plugin supports Android API level 21 and higher and minimum sdk version support is 21. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
+
+## Prerequisites
+
+Familiarity with Android Studio and the [fundamentals](https://developer.android.com/training/basics/firstapp) of android apps.
+
+## Basic Concepts
+
+-   `image background` - This is where the plugin will replace the video background with the image provided by user.
+-   `plugin load time` - This is the time taken by plugin to load ML model for the first time
+
+## Add dependency
+
+-   Add the 100ms Virtual Background plugin and sdk dependency to your app-level `build.gradle` [![](https://jitpack.io/v/100mslive/android-sdk.svg)](https://jitpack.io/#100mslive/android-sdk).
+
+```json{3,4}:build.gradle
+dependencies {
+ // See the version in the jitpack badge above.
+ implementation 'com.github.100mslive:android-sdk-virtual-background:x.x.x'
+ implementation 'com.github.100mslive:android-sdk:x.x.x'
+    }
+```
+##Interfaces
+
+### Instantiate VirtualBackground
+
+Instantiate the 100ms Virtual Background plugin like this:
+
+<Tabs id="instanstiate-VirtualBackground" items={['Kotlin', 'Java']} />
+
+<Tab id='instanstiate-VirtualBackground-0'>
+
+```kotlin
+      val hmsSdk = HMSSDK
+              .Builder(application)
+              .build()
+      val virtualBackground = HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+
+</Tab>
+
+<Tab id='instanstiate-VirtualBackground-1'>
+
+```java
+   HMSSDK hmssdk = new HMSSDK
+                           .Builder(application)
+                           .build();
+   val virtualBackground = new HMSVirtualBackground(hmssdk, imageBitmap)
+```
+</Tab>
+
+That's it. You have instanstiate virtual background plugin successfully. 🥳
+
+Now, let's take a look at the signature of the Join API
+
+As evident, `HMSVirtualBackground` accepts 2 arguments -
+-   `hmssdk`: an object of type `HMSSDK` class
+-   `imageBitmap`: A background image bitmap you want to replace your background to.
+
+## Check if plugin is supported
+
+isSupported can be used to check if the API level of device is supported or not.
+It will return True/False based on API Level.
+
+```kotlin
+if(virtualBackground.isSupported()){
+    console.log("Device is supported");
+}else{
+    console.log("Device not supported");
+}
+```
+
+## Init(Optional)
+
+Init is used to load model of virtual background for the first time.
+It takes on an average 300-500 ms.<br/>
+Calling init is handled internally by SDK if not done by user, in this case addPlugin call will
+take 300-500 ms for the first time and then less than 10ms in the subsequent calls.
```

Comment:
Consider throwing an exception if init is not called by the user because that needs to be called on a background thread and addPlugin need not be.

Reply:
I will update this